### DISCUSSION TITLE
Setting Default View for Grid

### DISCUSSION THREAD

**Discussion by cancan101** [pimcore/pimcore#16830] [Q&A / Support] [answered]
Is there a way to set the _default_ view for a given grid? i.e. to specify the set of columns and their ordering for all users on the system if they haven't then overridden the view themselves?

Looking at [the docs for Grid Configuration](https://pimcore.com/docs/pimcore/10.6/User_Documentation/DataObjects/Grid_Configuration_General.html), I see how I can share my view with others but not how I control the initial / default view for all users. 

We are trying to avoid having to ask all our users to click this:
![image](https://github.com/pimcore/pimcore/assets/51059/30a7972c-c117-4699-b81d-1b4f90a59d02)

**Comment by skoch98** [comment, marked as answer]
Hi, it's possible to define the predefined fields in the class definition. Each field has two checkboxes. One for the grid and one for the search result list.
For system default fields - like the Pimcore ID, Create Date & Co - there are checkboxes at the class-level in the definition.

^Sebastian

**Reply by cancan101** [reply]
When using the class definition "checkbox" approach, is there any way to control column ordering?

**Reply by skoch98** [reply]
Not as far as I know. Sorry.
